src/components/header.js

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'gatsby';
```

Comment:
This file should be deleted. It looks like the result of a bad merge with main.

Reply:
Oops, my bad.